add cwc2025 ro16 news recap (#13426)
ppy/osu-wiki: news/2025/2025-06-27-osucatch-world-cup-2025-round-of-16-recap.md

    <audio controls>
        <source src="https://assets.ppy.sh/artists/482/Songs/XenjeS - Planetary Exclusion Zone.mp3">
    </audio>
</div>

<style>
    .news-chat-quote__avatar {
        float: left;
        width: 40px;
        height: 40px;
        border-radius: 50%;
        margin-left: -50px;
    }

    .news-chat-quote__text-container {
        margin-left: 50px;
    }

    .news-chat-quote__username {
        font-weight: 600;
        margin-bottom: 2px;
    }

    .news-chat-quote__colour-white {
        color: #FFFFFF;
    }

</style>

This past week featured this year's first [osu! original](/wiki/osu!_originals)! [XenjeS](https://osu.ppy.sh/beatmaps/artists/482) made their CWC debut on [Planetary Exclusion Zone](https://osu.ppy.sh/beatmapsets/2388185#fruits/5168126), mapped by [-Rustyy](https://osu.ppy.sh/users/16355636) for the Nomod 2 slot. Xenje's blood-pumping electronic style made for the perfect fit for this week's technical pick, only enhanced by the work from our illustrators and graphic designers. Check them out by either clicking on the banner right above or [**here to watch the showcase video**](https://youtu.be/bjGpEYODFuc)!

> While the spectators and players only get to see the finished results for every original, a lot goes on behind the scenes for every single one of them. For this one, our head mappooler [wwwww](https://osu.ppy.sh/users/8434466) gave us some insights on the process for naming the track and and working on its background art.

Reviewer: ```suggestion
While the spectators and players only get to see the finished results for every original, a lot goes on behind the scenes for every single one of them. For this one, our head mappooler [wwwww](https://osu.ppy.sh/users/8434466) gave us some insights on the process for naming the track and working on its background art.
```

<div class="news-chat-quote__text-container">

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/8434466" style="background-image: url('/wiki/shared/avatars/wwwww.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-white" href="https://osu.ppy.sh/users/8434466">wwwww</a></p>

> [XenjeS](https://osu.ppy.sh/beatmaps/artists/482) didn’t know what to name the track, and let in me and [Yoshi_green](https://osu.ppy.sh/users/1035891) on coming up with name idea’s. We sent him back a list, in which he decided on yoshi’s idea of “Planetary Exclusion Zone or P.E.Z.”

Reviewer: ```suggestion
[XenjeS](https://osu.ppy.sh/beatmaps/artists/482) didn't know what to name the track, and let in me and [Yoshi_green](https://osu.ppy.sh/users/1035891) on coming up with name idea's. We sent him back a list, in which he decided on yoshi's idea of “Planetary Exclusion Zone or P.E.Z.”
```

> When Xenjes was asked about design idea’s, he gave the following answer 

Reviewer: ```suggestion
When Xenjes was asked about design ideas, he gave the following answer:
```

At this point, the maps were fairly close between the teams, with each having the potential to win essentially any pick depending on their performance. Canada barely managed to get on the board with their [HR2](https://osu.ppy.sh/beatmapsets/2388201#fruits/5168161) pick, only winning with a ridiculously small 775 points difference. Matches like these tend to be the most stressful ones for both the viewers and the players, because a single spin can make the difference. Canada could now breathe a bit as they had won the close point and secured their own pick. Surely no other pick will be this close...

France then went on to pick [NM3](https://osu.ppy.sh/beatmapsets/2388187#fruits/5168130) and once again set a top team-score, with ::{ flag=FR }:: [Electr0o](https://osu.ppy.sh/users/9484428) getting the only FC of the week on it. It was now up to Canada to pick, and they decided to go on [HR1](https://osu.ppy.sh/beatmapsets/2388200#fruits/5168160). And once again, on their second Hard Rock pick in a row, we would see a score difference of less than a thousand, with only 803 points of difference, but this time in favour of France. 

> But what would be the nail in the coffin for Canada was France's next pick, the final Hard Rock map, [HR3](https://osu.ppy.sh/community/matches/118474861). Even on purpose the match couldn't have been more tragic for Canada, as they would go on to lose this map by a whole 4 points, one of the smallest score differences across all World Cups combined. France was now on match point, and despite Canada's successful [HD1](https://osu.ppy.sh/beatmapsets/2388192#fruits/5168150) pick, they would manage to close out the match 6–2 on [NM4](https://osu.ppy.sh/beatmapsets/2388325#fruits/5168335) with another double FC from ::{ flag=FR }:: [Electr0o](https://osu.ppy.sh/users/9484428) and ::{ flag=FR }:: [Ttobas](https://osu.ppy.sh/users/2084568), who would end up being ranked #1 and #2 best performers of the round on the [statistics sheet](https://docs.google.com/spreadsheets/d/1X1yRy2JXnWeeZPFa1voLP7EPifW2sQ11XvB56wKQpv8/edit?rm=minimal&gid=2090207382#gid=2090207382).

Reviewer 2: probably saying that even if someone would have hypothetically rigged the match it wouldnt have been as tragic as it was in real life — losing to a 4 point difference

---
layout: post
title: "osu!catch World Cup 2025: Round of 16 Recap"
date: 2025-06-27 12:00:00 +0000
---

This year's Round of 16 is now behind us, and with it came the first of many eliminations to come.

![](/wiki/shared/news/2025-06-27-osucatch-world-cup-2025-round-of-16-recap/banner.jpg)

This first elimination round marked the end of the journey for 8 teams. The stakes are slowly rising as we progress in the bracket and approach the middle of the tournament, with a lot of action going on every week. Let's have a look at how things went!

## Mappool recap

<style>
    .osu-md--news .audio-player {
        max-width: inherit;
    }
</style>

**[Download the pool here! (140 MB)](https://osu.ppy.sh/beatmaps/packs/P289)**

<div align="center" class="osu-md__paragraph">

>     <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2486718794?t=5h8m30s&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>

Reviewer 3: ```suggestion
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2486718794&t=5h8m30s&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
```

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/482/Songs/XenjeS - Planetary Exclusion Zone.mp3">
    </audio>
</div>

<style>
    .news-chat-quote__avatar {
        float: left;
        width: 40px;
        height: 40px;
        border-radius: 50%;
        margin-left: -50px;
    }

    .news-chat-quote__text-container {
        margin-left: 50px;
    }

    .news-chat-quote__username {
        font-weight: 600;
        margin-bottom: 2px;
    }

    .news-chat-quote__colour-white {
        color: #FFFFFF;
    }

</style>

This past week featured this year's first [osu! original](/wiki/osu!_originals)! [XenjeS](https://osu.ppy.sh/beatmaps/artists/482) made their CWC debut on [Planetary Exclusion Zone](https://osu.ppy.sh/beatmapsets/2388185#fruits/5168126), mapped by [-Rustyy](https://osu.ppy.sh/users/16355636) for the No Mod 2 slot. XenjeS' blood-pumping electronic style made for the perfect fit for this week's technical pick, only enhanced by the work from our illustrators and graphic designers. Check them out by either clicking on the banner right above, or checking out the showcase video below!

> <div align="center">

Reviewer 3: ```suggestion
<div align="center" class="osu-md__paragraph">
```

| South Korea ::{ flag=KR }:: | 0 | **6** | ::{ flag=PE }:: **Peru** |
| Türkiye ::{ flag=TR }:: | 2 | **6** | ::{ flag=ES }:: **Spain** |
| Indonesia ::{ flag=ID }:: | 2 | **6** | ::{ flag=PL }:: **Poland** |
| Denmark ::{ flag=DK }:: | 0 | **6** | ::{ flag=TW }:: **Taiwan** |
| Vietnam ::{ flag=VN }:: | 3 | **6** | ::{ flag=HK }:: **Hong Kong** |
| **Italy** ::{ flag=IT }:: | **6** | 0 | ::{ flag=DE }:: Germany |
| **United States** ::{ flag=US }:: | **6** | 3 | ::{ flag=SE }:: Sweden |
| **Russian Federation** ::{ flag=RU }:: | **6** | 0 | ::{ flag=BE }:: Belgium |
| **Finland** ::{ flag=FI }:: | **6** | 1 | ::{ flag=GB }:: United Kingdom |
| **France** ::{ flag=FR }:: | **6** | 2 | ::{ flag=CA }:: Canada |

This Round of 16 didn't feature as many close matches as the Round of 32, but it still had some interesting ones as well as a few surprises. Only eight teams remain undefeated in the winners' bracket, and nearly all of them are from the top 8 of Qualifier seedings, save for ::{ flag=PL }:: Poland at seed 10. 4 of them will drop out of it next week in the Quarterfinals, so be on the lookout for some elite play from these teams next weekend!

> We only had a single upset this round, just like last week, with ::{ flag=PL }:: Poland (#10) triumphing over Indonesia ::{ flag=ID }:: (#7) in a convincing performance that resulted in a 6–2 win for them. Poland has had quite the roster change this year, losing some of their core players who managed to reach the podium last year. However, thanks to the sizeable pool of new talents being fostered in the country, the new roster still manages to perform and stay competitive in this edition. How far will their run bring them this year? It is all still left to be decided in the next couple of weeks!

Reviewer 3: ```suggestion
We only had a single upset this round, just like last week, with ::{ flag=PL }:: Poland (#10) triumphing over ::{ flag=ID }:: Indonesia (#7) in a convincing performance that resulted in a 6–2 win for them. Poland has had quite the roster change this year, losing some of their core players who managed to reach the podium last year. However, thanks to the sizeable pool of new talents being fostered in the country, the new roster still manages to perform and stay competitive in this edition. How far will their run bring them this year? It is all still left to be decided in the next couple of weeks!
```

**[You can view the match VOD here!](https://www.twitch.tv/videos/2492992095?t=0h9m34s)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&time=0h9m34s&video=2492992095" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

Looking at the winners' bracket matches last week, most people didn't anticipate a very exciting encounter between the United States (#1) and Sweden (#16). The mappool, even this early, wasn't easy enough for the teams way lower seeded than their opponents to go for an upset. Surely, last year's champions and #1 seed wouldn't have any issues with the 16th seed, right?

Last year, the United States reached the Grand Finals week with only 2 points lost during their winners' bracket run, beginning with their 7–1 win against Chile during the Semifinals. Sweden, on the other hand, finished in top 24 last year, being eliminated in the Quarterfinals. They had already struggled to maintain themselves in the winners' bracket last week, narrowly advancing after a tiebreaker win against Peru.

> Given all this, no one truly expected Sweden to be able to compete against the United States in this matchup. So it was to everyone's surprise that they went and won their first pick, [NM1](https://osu.ppy.sh/beatmapsets/2388183#fruits/5168115), with a double FC from ::{ flag=SE }:: [amni sd](https://osu.ppy.sh/users/16669841) and ::{ flag=SE }:: [Shameimaru](https://osu.ppy.sh/users/20340480). With that, the United States' perfect record was broken, with their flawless run unexpectedly coming to an end in this Round of 16.

Reviewer 3: ```suggestion
Given all this, no one truly expected Sweden to be able to compete against the United States in this matchup. It was thus to everyone's surprise that they went and won their first pick, [NM1](https://osu.ppy.sh/beatmapsets/2388183#fruits/5168115), with a double FC from ::{ flag=SE }:: [amni sd](https://osu.ppy.sh/users/16669841) and ::{ flag=SE }:: [Shameimaru](https://osu.ppy.sh/users/20340480). With that, the United States' perfect record was broken, with their flawless run unexpectedly coming to an end in this Round of 16.
```

The United States losing a point to Sweden, while unexpected, was also not something too shocking. With how good players have got, even lower-seeded teams are now able to compete and FC maps, leaving top teams with almost no margin for error. The United States brushed that off as a minor misstep and quickly refocused on their match, comfortably winning their next pick on [HR3](https://osu.ppy.sh/beatmapsets/2388197#fruits/5168156) with a second double FC in a row from ::{ flag=US }:: [Lexii](https://osu.ppy.sh/users/7226149) and ::{ flag=US }:: [Secre](https://osu.ppy.sh/users/2306637).

> Sweden tried again on their [MM1](https://osu.ppy.sh/beatmapsets/2388341#fruits/5168359) pick, but let the first breakpoint of the match slip away into the United States' favour. They were now down 2 points on their opponent's pick, and the match was going back to what one would expect of this matchup, a mostly one-sided victory from the United States. However, no one could have predicted what happened next.

Reviewer 3: ```suggestion
Sweden tried again on their [MM1](https://osu.ppy.sh/beatmapsets/2388341#fruits/5168359) pick, but let the first break point of the match slip away into the United States' favour. They were now down 2 points on their opponent's pick, and the match was going back to what one would expect of this matchup: a mostly one-sided victory from the United States. However, no one could have predicted what happened next.
```

> The United States selected [HD2](https://osu.ppy.sh/beatmapsets/2388193#fruits/5168151), the low AR Hidden pick, knowing what they're able to do on this map. ::{ flag=US }:: [Lexii](https://osu.ppy.sh/users/7226149) managed to get his third FC in a row and ::{ flag=US }:: [Dahcreeper](https://osu.ppy.sh/users/6926006) got his second, but Sweden answered back with a double FC of their own from ::{ flag=SE }:: [amni sd](https://osu.ppy.sh/users/16669841) and ::{ flag=SE }:: [Setomi79](https://osu.ppy.sh/users/2205929) to win the point. Winning a single point against the champions was something, but a second one, and on their own pick? This match was getting intense, and Sweden were performing to the very best of their abilities.

Reviewer 3: ```suggestion
The United States selected [HD2](https://osu.ppy.sh/beatmapsets/2388193#fruits/5168151), the low AR Hidden pick, knowing what they're able to do on this map. ::{ flag=US }:: [Lexii](https://osu.ppy.sh/users/7226149) managed to get his third FC in a row and ::{ flag=US }:: [Dahcreeper](https://osu.ppy.sh/users/6926006) got his second, but Sweden answered back with a double FC of their own from ::{ flag=SE }:: [amni sd](https://osu.ppy.sh/users/16669841) and ::{ flag=SE }:: [Setomi79](https://osu.ppy.sh/users/2205929) to win the point. Winning a single point against the champions was something, but a second one, and on their own pick? This match was getting intense, and the Swedish players were performing to the very best of their abilities.
```

> And it didn't stop there, with Sweden following up with mind-blowing 3-way FC on [NM2](https://osu.ppy.sh/beatmapsets/2388185#fruits/5168126) to win their own pick. It had been years since the last time the United States were contested like this so early in the bracket. However, champions don't become champions without strong mental resilience, and they went on to win the next 4 picks in a row, to close out the match 6–3. Even if they couldn't keep taking points from them, Sweden could be proud of their performance, as they gave the United States a battle that they're sure to remember for a while.

Reviewer 3: ```suggestion
And it didn't stop there, with Sweden following up with a mind-blowing 3-way FC on [NM2](https://osu.ppy.sh/beatmapsets/2388185#fruits/5168126) to win their own pick. It had been years since the last time the United States were contested like this so early in the bracket. However, champions don't become champions without strong mental resilience, and they went on to win the next 4 picks in a row, to close out the match 6–3. Even if they couldn't keep taking points from them, Sweden could be proud of their performance, as they gave the United States a battle that they're sure to remember for a while.
```
